text/machine_comprehension/roberta/README.md

```diff
@@ -0,0 +1,88 @@
+# RoBERTa
+
+## Use cases
+Transformer-based language model for text generation.   
+
+## Description
+RoBERTa builds on BERT’s language masking strategy and modifies key hyperparameters in BERT, including removing BERT’s next-sentence pretraining objective, and training with much larger mini-batches and learning rates. RoBERTa was also trained on an order of magnitude more data than BERT, for a longer amount of time. This allows RoBERTa representations to generalize even better to downstream tasks compared to BERT.
+
+## Model
+
+ |Model        |Download  |Download (with sample test data)| ONNX version |Opset version|Accuracy|
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
+|RoBERTa-BASE| [499 MB](model/roberta-base-11.onnx) |  [295 MB](model/roberta-base-11.tar.gz) |  1.6 | 11|MCC of [0.85](dependencies/roberta-sequence-classification-validation.ipynb)|
+|RoBERTa-SequenceClassification| [501 MB](model/roberta-sequence-classification-9.onnx) |  [434 MB](model/roberta-sequence-classification-9.tar.gz) |  1.4 | 9|88.5|
+
+## Source
+PyTorch RoBERTa => ONNX RoBERTa
+PyTorch RoBERTa + script changes => ONNX RoBERTa-SequenceClassification
+
+## Conversion
+Here is the [benchmark script](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/transformers/run_benchmark.sh) that was used for exporting RoBERTa-BASE model. 
+
+Tutorial for conversion of RoBERTa-SequenceClassification model can be found in the [conversion](https://github.com/SeldonIO/seldon-models/blob/master/pytorch/moviesentiment_roberta/pytorch-roberta-onnx.ipynb) notebook.
+
+Official tool from huggingface that can be used to convert transformers models to ONNX can be found [here](https://github.com/huggingface/transformers/blob/master/src/transformers/convert_graph_to_onnx.py)
+
+## Inference
+We used [ONNX Runtime](https://github.com/microsoft/onnxruntime) to perform the inference.
+
+Tutorial for running inference for RoBERTa-SequenceClassification model using onnxruntime can be found in the [inference](dependencies/roberta-inference.ipynb) notebook.
+
+### Input
+Input is a sequence of words as a string. Example: "This film is so good", tokenized by RobertaTokenizer. input_ids: Indices of input tokens in the vocabulary. It's a int64 tensor of dynamic shape (batch_size, sentence_length).
```

Comment:
```suggestion
Input is a sequence of words as a string. Example: "This film is so good", tokenized by RobertaTokenizer. 

- input_ids: Indices of input tokens in the vocabulary, an int64 tensor of dynamic shape (batch_size, sentence_length).
```

Is input_ids the only input here? Does this differ for roberta-base / sequence-classification? Might be better to clarify.